docs/wfe.rst

```diff
@@ -0,0 +1,216 @@
+Representing sources of wavefront error
+=======================================
+
+When used for Fraunhofer-domain propagation, POPPY supports expressing aberrations as if they were applied in a pupil plane. Currently, there is the `~poppy.ZernikeWFE` optical element to represent wavefront error as a combination of Zernike terms, and the `~poppy.ParameterizedWFE` optical element that offers the ability to specify basis sets other than Zernikes.
```

Comment:
The first sentence should be rephrased - these are equally applicable in a Fresnel propagation as in Fraunhofer. There's no restriction on pupil vs image plane either. Zernikes are a basis set for parameterizing wavefront error over a circular aperture on some surface, regardless of what optical conjugate that surface might be at.   

E.g. it's totally correct to parameterize wavefront error over JWST's secondary mirror with a Zernike basis, even though that mirror is not at a pupil conjugate.